### PR TITLE
US124277 - Clean-up languages

### PR DESCRIPTION
**Pull request by devpow112**
Move languages into similar location as other repos and add missing languages.

### CODE REVIEW

**Comment by devpow112** [comment]
Trying to resolve #33 first. The generated translations are bad in that PR so seeing if they can re-run it to hopefully fix the issue.

**Comment by devpow112** [comment]
> Trying to resolve #33 first. The generated translations are bad in that PR so seeing if they can re-run it to hopefully fix the issue.

Just ended up integrating the changes in it into this PR with fixes.

File: langtools/config.json [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
     "copySourceLang": "en",
-    "langDir": "serge",
+    "langDir": "lang",
     "langNames": [
         "ar",
+        "cy",
         "da",
         "de",
         "en",
+        "es-es",
```
Comment (mpharoah-d2l) [inline langtools/config.json:12]: Should there also be one for `es-mx`?

Comment (devpow112) [reply langtools/config.json:12]: `es-mx` is represented by just `es`

**Review by mpharoah-d2l — APPROVED** [review]

**Pull request merged**